src/Snackbar/Snackbar.react.js

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable import/no-unresolved, import/extensions */
+import React, { PropTypes, PureComponent } from 'react';
+import { View, Text, Animated, Easing, Platform, StyleSheet } from 'react-native';
+
+import Button from '../Button';
+
+const propTypes = {
+    /**
+    * The text message to display.
+    */
+    message: PropTypes.string.isRequired,
+    /**
+    * Whether or not the snackbar is visible.
+    */
+    visible: PropTypes.bool.isRequired,
+    /**
+    * The amount of time in milliseconds to show the snackbar.
+    */
+    timeout: PropTypes.number.isRequired,
+    /**
+    * Callback for when the timeout finishes.
+    */
+    onRequestClose: PropTypes.func.isRequired,
+    /**
+    * Whether or not there is a bottom navigation on the screen.
+    */
+    bottomNavigation: PropTypes.bool.isRequired,
+    /**
+    * The function to execute when the action is clicked.
+    */
+    actionHandler: PropTypes.func,
+    /**
+    * The function to execute when the action is clicked.
+    */
+    actionText: PropTypes.string,
+    /**
+    * Inline style of snackbar
+    */
+    style: PropTypes.shape({
+        container: View.propTypes.style,
+        message: View.propTypes.style,
+        actionContainer: View.propTypes.style,
+        actionText: View.propTypes.style,
+    }),
+};
+const defaultProps = {
+    visible: false,
+    timeout: 2750,
+    bottomNavigation: false,
+    style: {},
+};
+const contextTypes = {
+    uiTheme: PropTypes.object.isRequired,
+};
+
+function getStyles(props, context) {
+    const { snackbar } = context.uiTheme;
```

Comment:
It's true probably, but I think it doesn't matter ... we'll use flat design ... we don't have to use another styles of button ... And I think it's always better to reuse one code then make duplicity ..

-----
And I think there will be someone in future who wants raised button on snackbar :D